### ISSUE TITLE
Breadcrumbs still shown with "zenMode.showTabs": "none"

### ISSUE THREAD

**Issue by brussee** [microsoft/vscode#216822]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.90.2
- OS Version: Windows 11 Pro

Steps to Reproduce:

1. Open any file in the editor
2. Toggle Zen-mode on

**Comment by brussee** [comment]
Related: https://github.com/microsoft/vscode/pull/195222

**Comment by jrieken** [comment]
Sorry @isidorn but I don't know who own zen mode

**Comment by isidorn** [comment]
I see zen-mode issues assigned to SteVen so will go ahead and assign to him. SteVen feel free to assign back to me.

**Comment by benibenj** [comment]
`zenMode.showTabs` does not influence if breadcrumbs are shown, similar to `workbench.editor.showTabs`.
You can hide breadcrumbs with the `breadcrumbs.enabled` setting. Is a zen mode `zenMode.breadcrumbsEnabled` setting really needed here?

**Comment by brussee** [comment]
> `zenMode.showTabs` does not influence if breadcrumbs are shown, similar to `workbench.editor.showTabs`. You can hide breadcrumbs with the `breadcrumbs.enabled` setting. Is a zen mode `zenMode.breadcrumbsEnabled` setting really needed here?

tl;dr: Yes.

Zen Mode currently has settings to: Hide Activity Bar, Hide Line Numbers, Hide Status Bar, (all enabled by default) among other settings.
The only thing missing is: Hide Breadcrumbs (enabled by default).

It's the only element remaining remaining before reaching true code zen: having only code editor on the screen and nothing else: no distractions like navigation breadcrumbs.

Thanks in advance for your effort!

**Comment by jumble** [comment]
+1, this would be a nice addition

**Comment by doehyunbaek** [comment]
I made workaround not perfect but good enough for me. Use keybinding to toggle both the zen mode and breadcrumbs.
Though I think the addition would be really nice!

```json
[
    {
        "key": "ctrl+cmd+f",
        "command": "runCommands",
        "when": "!inZenMode",
        "args": {
            "commands": [
                "breadcrumbs.toggle",
                "workbench.action.toggleZenMode"
            ]
        }
    },
    {
        "key": "ctrl+cmd+f",
        "command": "runCommands",
        "when": "inZenMode",
        "args": {
            "commands": [
                "workbench.action.toggleZenMode",
                "breadcrumbs.toggle"
            ]
        }
    }
]
```
